### ISSUE TITLE
adultmult.club (tv.cartoonka.art, pro.multmania.club, braindead.me)

### ISSUE THREAD

**Issue by EugeneRymarev** [ytdl-org/youtube-dl#31439]
## Checklist

- [x] I'm reporting a new site support request
- [x] I've verified that I'm running youtube-dl version **2021.12.17**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that none of provided URLs violate any copyrights
- [x] I've searched the bugtracker for similar site support requests including closed ones

## Example URLs

- Single video: https://tv.cartoonka.art/animation/strange_world.html
- Single video: https://pro.multmania.club/other/strange_world.html
- Single video: https://braindead.me/featurefilm/akira.html
- Playlist: https://tv.cartoonka.art/tvseries/litvinenko.html
- Playlist: https://braindead.me/tvseries/american-gods.html
- Playlist: https://tv.cartoonka.art/animation/aquaman_king_of_atlantis.html
- Playlist: https://tv.cartoonka.art/anime/dota_dragons_blood.html

## Description

Movies and TV series dubbed and original

**Comment by dirkf** [comment]
On first look, I see the movie link in the first example is geo-restricted to (presumably) Russia: at least, "content is not available in your region".

**Comment by EugeneRymarev** [comment]
> geo-restricted

@dirkf, Yes, but I use prostovpn's antizapret on my router and it works great for me.

**Comment by dirkf** [comment]
The other domains don't seem to be related. Please open separate site support requests for this
>* Single video: https://pro.multmania.club/other/strange_world.html

and for this
>* Single video: https://braindead.me/featurefilm/akira.html
>* Playlist: https://braindead.me/tvseries/american-gods.html
